src/crl.c

```diff
@@ -1562,15 +1565,28 @@ int LoadCRL(WOLFSSL_CRL* crl, const char* path, int type, int monitor)
             }
         }
 
+#ifndef CRL_REPORT_LOAD_ERRORS
         if (!skip && ProcessFile(NULL, name, type, CRL_TYPE, NULL, 0, crl,
                                  VERIFY) != WOLFSSL_SUCCESS) {
             WOLFSSL_MSG("CRL file load failed, continuing");
         }
+#else
+        if (!skip) {
+            int temp = ProcessFile(NULL, name, type, CRL_TYPE, NULL, 0, crl,
```

Comment:
Consider using `ret` instead of creating a new `int temp`. Everything else looks good.